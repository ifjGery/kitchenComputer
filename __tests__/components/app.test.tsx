import * as React from 'react';
import * as renderer from 'react-test-renderer';
import App from '../../packages/ui-components/app/index';

test('Hello world is displayed', () => {
  const component = renderer.create(<App />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Two number is equal', () => {
  expect(2 + 2).toBe(4);
});
