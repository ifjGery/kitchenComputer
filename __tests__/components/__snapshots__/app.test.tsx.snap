// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello world is displayed 1`] = `
<div
  className="app"
>
  <h1>
     Hello, World! 
  </h1>
</div>
`;
